cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/domain/CloudBuild.java

```diff
@@ -1,22 +1,27 @@
 package org.cloudfoundry.client.lib.domain;
 
+import java.util.Date;
 import java.util.List;
 import java.util.UUID;
 
 public class CloudBuild extends CloudEntity {
 
     private CreatedBy createdBy;
-    
+
     private Droplet droplet;
-    
+
     private String error;
-    
+
     private Lifecycle lifecycle;
-    
+
     private Package packageInfo;
-    
+
     private BuildState state;
-    
+
+    private Date createdAt;

```

Comment:
These fields already exist in the `meta` field that is inherited from the parent class.